### ISSUE TITLE
image_to_pdf_or_hocr causing FileNotFoundError

### ISSUE THREAD

**Issue by jhonratiu** [madmaze/pytesseract#257]
This function is causing the following error on linux

```
~/.local/lib/python3.7/site-packages/pytesseract/pytesseract.py in run_and_get_output(image, extension, lang, config, nice, timeout, return_bytes)
    257         run_tesseract(**kwargs)
    258         filename = kwargs['output_filename_base'] + extsep + extension
--> 259         with open(filename, 'rb') as output_file:
    260             if return_bytes:
    261                 return output_file.read()

FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tess__qw1spg8.hocr'
```
Both image_to_data and image_to_string work fine, outputting the correct data.

**Comment by int3l** [comment]
Hi @jhonratiu , what is the version of pytesseract?
Can you update to the latest available one (0.3.2) and try again?

**Comment by jhonratiu** [comment]
Worked it out by testing tesseract directly. Error was due to overriding the configs folder in tessdata when downloading data files to it. Thanks.
